installer.sh

```diff
@@ -811,24 +830,31 @@ if [ "$HOST_PLATFORM" = "linux" ]; then
 elif [ "$HOST_PLATFORM" = "freebsd" ] || [ "$HOST_PLATFORM" = "osx" ]; then
 	IOB_BIN_PATH=/usr/local/bin
 fi
+
 # Symlink the global binaries iob and iobroker
+$SUDOX ln -sfn $IOB_DIR/iobroker $IOB_BIN_PATH/iobroker
+
+# ADOE: probably wrong? (iob   vs   iobroker)
 if [ "$IS_ROOT" = true ]; then
-	ln -sfn $IOB_DIR/iobroker $IOB_BIN_PATH/iobroker
-	ln -sfn $IOB_DIR/iobroker $IOB_BIN_PATH/iob
+	     ln -sfn $IOB_DIR/iobroker $IOB_BIN_PATH/iob
 else
-	sudo ln -sfn $IOB_DIR/iobroker $IOB_BIN_PATH/iobroker
-	sudo ln -sfn $IOB_DIR/iob $IOB_BIN_PATH/iob
+	sudo ln -sfn $IOB_DIR/iob      $IOB_BIN_PATH/iob
 fi
```

Comment:
The root version was correct, so it should be
```
$SUDOX ln -sfn $IOB_DIR/iobroker $IOB_BIN_PATH/iobroker
$SUDOX ln -sfn $IOB_DIR/iobroker $IOB_BIN_PATH/iob
```
`$IOB_DIR/iobroker` is the actual executable, `$IOB_DIR/iob` is just linked there like the ones in `IOB_BIN_PATH`

Reply:
corrected in INST and FIX